projects/ngx-d3timeline/src/lib/content/content.service.ts

```diff
@@ -6,25 +6,17 @@ import { TimelineEvent } from '../timeline-event';
 import { EventRectangle } from './content';
 import { Orientation } from '../orientation';
 import { TimeScale, BandScale } from '../scale-types';
-import { combineLatest } from 'rxjs';
-import { DragService } from './drag.service';
 import { TimelineDragEvent } from './timeline-drag-event';
 
 @Injectable({ providedIn: 'root' })
 export class ContentService {
-  eventRectangles$ = combineLatest([
-    this.store.state$,
-    this.dragService.drag$
-  ]).pipe(
-    map(([scales, dragEvent]) => this.createEventRectangles(scales, dragEvent))
+  eventRectangles$ = this.store.state$.pipe(
+    map(this.createEventRectangles.bind(this))
```

Comment:
more easy to follow:

```suggestion
    map(state => this.createEventRectangles(state))
```
but moot if we use the selectors I did

Reply:
done